File: src/002/main.c
# include <stdio.h>

int main () {
	unsigned int sum = 0;
	unsigned int a = 1;
	unsigned int b = 1;
	
	while (b < 4000000) {
		unsigned int c = a;
		a = b;
		b += c;
		
		if (a % 2 == 0) {
			sum += a;
		}
	}
	printf ("%u\n", sum);
}

File: src/002/main.rs

fn main () {
	let mut sum: u32 = 0;
	let mut a: u32 = 1;
	let mut b: u32 = 1;
	
	while b < 4_000_000 {
		let c: u32 = a;
		a = b;
		b += c;
		
		if a % 2 == 0 {
			sum += a;
		}
	}
	
	println! ("{}", sum);
}

